katas/content/superposition/three_states_two_qubits_phases/solution.md

```diff
@@ -0,0 +1,20 @@
+To start, we will prepare the $\frac{1}{\sqrt{3}} \big( |00\rangle +  |01\rangle + |10\rangle \big)$ state using the solution to task 2.3. To get to the final state, we need to add the relative phases to both $|01\rangle$ and $|10\rangle$ basis states without changing the $|00\rangle$ state.
+
+First, we want to transform the $|01\rangle$ state to the $\omega |01\rangle = e^{2\pi i/3} |01\rangle$ state, while not changing the other states.
+Using the [$R_1$](../tutorials/SingleQubitGates/SingleQubitGates.ipynb#Rotation-Gates) gate, we can change a qubit state from $|1\rangle$ to $e^{i\theta}|1\rangle$ without changing the $|0\rangle$ state.
```

Comment:
I left this link in but I presume it needs updating, though I wasn't sure where to: `../tutorials/SingleQubitGates/SingleQubitGates.ipynb#Rotation-Gates`

Reply:
R1 gate documentation is here. https://learn.microsoft.com/qsharp/api/qsharp-lang/microsoft.quantum.intrinsic/r1.

I checked that Single Qubit Gates does not contain example for R1 gate.

Reply:
In general, new QDK api are at : https://learn.microsoft.com/qsharp/api/. You would need to explore the relevant API by drilling down in the relevant namespace.
![image](https://github.com/microsoft/qsharp/assets/40084144/b332e772-e927-4d60-b094-7fd74c66bdc6)